static/app/views/performance/onboarding.tsx

```diff
@@ -101,6 +107,54 @@ function Onboarding({organization}: Props) {
       duration,
     });
   }
+  const showSampleTransactionBtn = organization.features.includes(
+    'performance-create-sample-transaction'
+  );
+  const featureTourBtn = (
+    <FeatureTourModal
+      steps={PERFORMANCE_TOUR_STEPS}
+      onAdvance={handleAdvance}
+      onCloseModal={handleClose}
+      doneUrl={performanceSetupUrl}
+      doneText={t('Start Setup')}
+    >
+      {({showModal}) => (
+        <Button
+          priority={showSampleTransactionBtn ? 'link' : 'default'}
+          onClick={() => {
+            trackAnalyticsEvent({
```

Comment:
@Neo-Zhixing mind updating to the `trackAdvancedAnalyticsEvent` function heree? We can make a new file for Performance events (see static/app/utils/issueEvents.tsx as an example).